### DISCUSSION TITLE
Blockchain, smart contract, solidity

### DISCUSSION THREAD

**Discussion by Ugochi-Eze** [smartcontractkit/full-blockchain-solidity-course-py#1512] [General]
I am studying building construction, I picked interest in writing about recycling of construction and demolition waste on my project. In my research I came across sustainable ways of managing construction and demolition waste, recycling which is part of it. In the course of my research I found out how blockchain technology has  being use to manage construction and demolition waste , which got my interest in learning more about blockchain.

My aim of learning blockchain, building smart contract is to apply them in the construction industry.
My focus area is to imbed blockchain based platform in project execution in the construction industry.

**Comment by cromewar** [comment]
With that focus on mind you should take a look about blockchain applied to supply chain, is there anything you want help with?

**Comment by Ugochi-Eze** [comment]
Thank you
